sufia-models/app/models/concerns/sufia/works/generic_work.rb

```diff
@@ -11,5 +11,7 @@ module GenericWork
     include Sufia::GenericFile::Batches
     include Sufia::GenericFile::Content
     include Sufia::GenericFile::Permissions
+    include Sufia::GenericFile::Metadata
```

Comment:
This is not valid as we are already including the Work Metadata.  Most likely a merge conflict.